chart/kubeapps/templates/kubeops-rbac.yaml

```diff
@@ -79,5 +79,42 @@ subjects:
     name: {{ template "kubeapps.kubeops.fullname" . }}
     namespace: {{ .Release.Namespace }}
 {{- end -}}
+{{- if .Values.featureFlags.operators }}
```

Comment:
Do we need the same RBAC for tiller-proxy? Guess we can just limit this to helm3 for now anyway?

Also, is this RBAC is only required now because we're directly querying the operators? (as opposed to querying the assetsvc at a later date).

Reply:
true, I forgot to add this for tiller-proxy